.cirrus.yml

```diff
@@ -77,6 +121,27 @@ task:
       test_script:
         - cd $INTEGRATION_TEST_PATH/example/
         - flutter drive -v --driver=test_driver/integration_test.dart --target=integration_test/example_test.dart -d web-server --release --browser-name=chrome
+
+# Heavy-workload tasks.
+# These use machines with more CPUs and memory, so will reduce parallelization
+# for non-credit runs.
+task:
```

Comment:
does it need `cpu` and `memory` values under `container`?

Reply:
No, there are defaults (2 CPUs and 4 GB)

Reply:
I was wondering what makes these tasks use machines with more CPUs and memory as mentioned in the comment above?

Reply:
🤦 

I didn't look carefully enough at the context and thought this was about the light-load config. Fix incoming.